package.json

```diff
@@ -154,14 +154,13 @@
     "gulp": "^3.9.1",
     "gulp-cli": "^2.0.1",
     "gulp-postcss": "^7.0.1",
-    "gulp-sass": "^3.1.0",
+    "gulp-sass": "4.0.1",
     "gulp-sourcemaps": "^2.6.1",
     "husky": "^0.14.3",
     "jest": "23.4.1",
     "lint-staged": "7.2.0",
     "livereload": "^0.7.0",
     "node-mocks-http": "^1.7.0",
-    "node-sass": "^4.5.3",
```

Comment:
Removing this because `gulp-sass` already install a version of this as a dependency so avoids doubling up.

Reply:
I remember upgrading this caused some build issues a few months ago (something to do with how the new `node-sass` interpreted our media queries) so let's keep an eye on this when we deploy it, though it's probably fine.